src/stream-ciphers.rst

```diff
@@ -30,11 +30,11 @@ stream into different blocks, and encrypt each block:
    \overbrace{\mathtt{APOHGMMW}} & \overbrace{\mathtt{PVMEHQOM}} & \overbrace{\mathtt{MEEZSNFM}} & ...
    \end{matrix}
 
-This scheme is called :term:`ECB mode` (Electronic Code Book Mode), and it is
-one of the many ways that block ciphers can be used to construct stream
+The scheme is known as :term:`ECB mode` (Electronic Code Book Mode). It is
+one of several methods in which block ciphers can construct stream
```

Comment:
I appreciate removing a passive here, but it seems kind of strange to think of a mathematical algorithm as "constructing" anything.